### PR TITLE
CSS Changes Home View

### PR DESCRIPTION
**Pull request by farooq029**
CSS Changes of main pages image view

### CODE REVIEW

**Comment by paulatorre** [comment]
Hi! To add a search bar I had to modify the home view, add some new components and adjust the CSS. So I think I got this covered in my pull request (and with better BEM notation :P)

**Comment by farooq029** [comment]
Yes sure

File: src/components/HomeUsersView/index.css [hunk 1]
```diff
@@ -1,10 +1,10 @@
 .img__user {
```
Comment (VernitaJ) [inline src/components/HomeUsersView/index.css]: Can you change this to home-view__img

File: src/components/HomeUsersView/index.css [hunk 2]
```diff
@@ -38,3 +38,6 @@ ul {
   justify-content: space-between;
   margin-bottom: 5px;
 }
+.users_name{
```
Comment (VernitaJ) [inline src/components/HomeUsersView/index.css]: Change to BEM - home-view__user-name

File: src/components/HomeUsersView/index.css (hunk 1 above)
Comment (farooq029) [reply src/components/HomeUsersView/index.css]: Fixed

File: src/components/HomeUsersView/index.css (hunk 2 above)
Comment (farooq029) [reply src/components/HomeUsersView/index.css]: Fixed

File: src/components/HomeUsersView/Users.jsx [hunk 3]
```diff
@@ -4,10 +4,10 @@ import "./index.css";
 
 const Users = ({ userId, name, surname, profilePic }) => (
   <div className="view__users">
-    <Link to={`/${userId}`}>
+    <Link className="home-view__user-name" to={`/${userId}`}>
```
Comment (VernitaJ) [inline src/components/HomeUsersView/Users.jsx]: Here the home-view__user-name is used correctly. We can merge this now.

**Pull request merged**